saleor/graphql/csv/sorters.py

```diff
@@ -7,7 +7,8 @@ class ExportFileSortField(graphene.Enum):
     STATUS = ["status"]
     CREATED_AT = ["created_at"]
     UPDATED_AT = ["updated_at"]
-    CREATED_BY = ["created_by__id"]
+    USER = ["user__id"]
```

Comment:
Maby sort by `user__email`, `app_name`?

Reply:
`app__name` sounds good, but I am not totally convinced about sorting by user email. Maybe first user `last_name` and then `first_name`

Reply:
I was thinking if we need sorting by user/app at all, we don't have any designs for that yet so it would be fine for me to no add it all.

Reply:
That's true. So I'll remove that.